Providers/ProfilePrefs.swift

```diff
@@ -87,7 +87,11 @@ public class NSUserDefaultsProfilePrefs: Prefs {
     }
 
     public func stringArrayForKey(defaultName: String) -> [String]? {
-        return userDefaults.stringArrayForKey(qualifyKey(defaultName)) as [String]?
+        let objects = userDefaults.stringArrayForKey(qualifyKey(defaultName))
+        if let strings = objects as? [String] {
+            return strings
+        }
+        return nil
```

Comment:
No; a guy on the Swift compiler team asked me to file a Radar about that.

https://gist.github.com/rnewman/541a558f143cc64d232f